src/Illuminate/Database/Connectors/PostgresConnector.php

```diff
@@ -81,34 +81,42 @@ protected function configureTimezone($connection, array $config)
     }
 
     /**
-     * Set the schema on the connection.
+     * Set the search_path on the connection.
      *
      * @param  \PDO  $connection
      * @param  array  $config
      * @return void
      */
-    protected function configureSchema($connection, $config)
+    protected function configureSearchPath($connection, $config)
     {
-        if (isset($config['schema'])) {
-            $schema = $this->formatSchema($config['schema']);
+        if (isset($config['searchpath']) || isset($config['schema'])) {
+            $searchPath = $this->formatSearchPath($config['searchpath'] ?? $config['schema'] ?? '"public"');
```

Comment:
@cbj4074 since you introduced `->formatSearchPath()`, doesn't this mean the last null-coalesce default value should be simply `'public'` instead of `'"public"'`, as the method will take care of it?

Reply:
You may be right. It will be a few minutes but I'll fire up the laptop and confirm this.

Reply:
@mfn Tests do check out in either case. Regarding tests they explicitly check a literal string which tells me in both cases the string is formatted correctly.

tests/Database/DatabaseConnectorTest.php
```php
        $connection->shouldReceive('prepare')->once()->with('set search_path to "public"')->andReturn($connection);
```

Reply:
In fact we could remove the `?? '"public"'` altogether and tests will pass as we explicitly check that either `schema` OR `searchpath` isset ... and tests would still pass